packages/sysutils/ntfs-3g_ntfsprogs/package.mk

```diff
@@ -1,4 +1,4 @@
-################################################################################
+7################################################################################
```

Comment:
What the hell... I thought I understood the system... and then it strikes again against me. :-)
Nevertheless the one provided by you is correct.